test/unit/model/installable-item-test.js

```diff
@@ -232,11 +232,25 @@ describe('InstallableItem', function() {
   });
 
   describe('setOptionLocation method', function() {
-    it('should set location property for specific option', function() {
+    it('should set location property for specific option if it is present', function() {
       item.addOption('detected');
       item.setOptionLocation('detected', 'location');
       expect(item.option.detected.location).to.be.equal('location');
     });
+
+    it('should set location property for specific option if it is present', function() {
```

Comment:
You are correct, that is supposed to test if branch for case when option is not present. I'll fix it, thanks!